packages/backend/src/api/resources/ExternalAccount.ts

```diff
@@ -28,7 +28,7 @@ export class ExternalAccount {
       data.email_address,
       data.first_name,
       data.last_name,
-      data.image_url,
+      data.image_url || '',
```

Comment:
I think it will always return a string (even of failures) it will return an empty string. We should update the OpenAPI spec and the BAPI/FAPI serializers (drop the `omitempty`).